features/upgrade/workloads/descheduler-upgrade.feature

```diff
@@ -16,6 +16,7 @@ Feature: Descheduler major upgrade should work fine
     When I run the :create admin command with:
       | f | kubedescheduler-<%= cb.master_version %>.yaml |
     Then the step should succeed
+    Given 60 seconds have passed
```

Comment:
Hard wait should always avoid unless necessary and can not be workaround.
@pruan-rht  Thoughts ?

Reply:
@liangxia i do not think there is a way for me to workaround since i wanted this test to work for all 4.6 to 4.10 clusters, in 4.10 there has been a fix where the cluster pod comes up, terminates and a new cluster pod comes up and that is the reason i have put a hard wait.

Reply:
@kasturinarra why the wait makes a difference?  In the current repo I see 
```
    Then the step should succeed
    And status becomes :running of exactly 1 pods labeled:
      | app=descheduler |
 ```
 perhaps try to use this step instead  `Given /^a pod becomes ready with labels:$/ `

Reply:
@pruan-rht the problem is a pod comes up with label app=descheduler then runs for around 7seconds and another pod comes up with same label app=descheduler and continues to be running state. The time it stays is very less and the same case needs to be executed for all other versions where in which this change is not present.(like pod coming, disappearing and another pod coming up and running). So i feel having a hard wait will avoid seeing failures with respect to the case and that is the reason for it.

Reply:
@pruan-rht could you please help check my reply and review this again ? thanks !!

Reply:
@kasturinarra ok, let's see if this will make it more stable.
/lgtm